pmpro-approvals.php

```diff
@@ -1073,17 +1073,17 @@ public static function pmpro_account_bullets_top() {
 	 */
 	public static function pmpro_members_list_user( $user ) {
 
-		// Hide ('pending') link from the following statuses.
-		$status_in = apply_filters( 'pmpro_approvals_members_list_status', array( 'oldmembers', 'cancelled', 'expired' ) );
-		$level_type = isset( $_REQUEST['l'] ) ? $_REQUEST['l'] : '';
+	// Hide ('pending') link from the following statuses.
+	$status_in = apply_filters( 'pmpro_approvals_members_list_status', array( 'oldmembers', 'cancelled', 'expired' ) );
+	$level_type = isset( $_REQUEST['l'] ) ? $_REQUEST['l'] : '';
 
-		if ( current_user_can( 'pmpro_approvals' ) && self::isPending( $user->ID, $user->membership_id ) && ! in_array( $level_type, $status_in ) ) {
-			$user->membership .= ' (<a href="' . admin_url( 'admin.php?page=pmpro-approvals&s=' . urlencode( $user->user_email ) ) . '">' . __( 'Pending', 'pmpro-approvals' ) . '</a>)';
-		}
-
-		return $user;
+	if ( isset( $_REQUEST['page']) && $_REQUEST['page'] === 'pmpro-dashboard' ) && current_user_can( 'pmpro_approvals' ) && self::isPending( $user->ID, $user->membership_id ) && ! in_array( $level_type, $status_in ) ) {
```

Comment:
There's a syntax error with line 1080. There's a closing `)` just after 'pmpro-dashboard'

Reply:
I have fixed this :)